uber/configspec.ini

```diff
@@ -7,6 +7,10 @@
 # after a deadline or on-site.
 dev_box = boolean(default=True)
 
+# Redirect 404s to Uber's default URL
+default_url = string(default="/uber/common/")
+default_url_priority = integer(default=1)
```

Comment:
Hm, don't we already have this in the form of `path` and `url_root` and `url_base`?